helm/publisher/templates/worker/deployment.yaml

```diff
@@ -0,0 +1,90 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Release.Name }}-worker
+  labels:
+    app: {{ .Release.Name }}-worker
+spec:
+  replicas: {{ .Values.workerReplicaCount}}
+  selector:
+    matchLabels:
+      app: {{ .Release.Name }}-worker
+  template:
+    metadata:
+      labels:
+        app: {{ .Release.Name }}-worker
+      annotations:
+        # TODO: if Helm library is used, `$.Template.BasePath` needs to be replaced
+        # by chart name, see https://helm.sh/docs/howto/charts_tips_and_tricks/#automatically-roll-deployments
+        checksum/configmap: {{ include (print $.Template.BasePath "/common/configmap.yaml") . | sha256sum }}
+        checksum/secrets: {{ include (print $.Template.BasePath "/common/secrets.yaml") . | sha256sum }}
+    spec:
+      containers:
+        - name: app
+          image: "{{ .Values.appImage.repository }}:{{ .Values.appImage.tag }}"
+          command: ["foreman"]
+          args: ["run", "worker"]
+          ports:
+            - name: http
+              containerPort: {{ .Values.appPort }}
+          envFrom:
+            - configMapRef:
+                name: {{ .Release.Name }}-env
+          env:
+          {{- range .Values.secrets }}
+            - name: {{ . | title }}
+              valueFrom:
+                secretKeyRef:
+                  name: {{ $.Release.Name }}
+                  key: {{ . | title }}
+          {{- end }}
+          {{- with .Values.webExtraEnv }}
+            {{- . | toYaml | trim | nindent 12 }}
+          {{- end }}
+          resources:
+{{ toYaml .Values.webResources | indent 12 }}
+          livenessProbe:
+            exec:
+              command:
+              - /bin/sh
+              - -c
+              - exit 0
+            initialDelaySeconds: 30
+            periodSeconds: 5
+          readinessProbe:
+            exec:
+              command:
+              - /bin/sh
+              - -c
+              - exit 0
+            initialDelaySeconds: 30
+            periodSeconds: 5
+        - name: nginx
```

Comment:
Presumably a worker process also doesn't need an nginx proxy container?

Reply:
I can remove it, I guess maybe we don't need.